package.json

```diff
@@ -31,7 +31,12 @@
     "body-parser": "^1.17.2",
     "bootflat": "^2.0.4",
     "chalk": "^2.0.1",
+    "cheerio": "^1.0.0-rc.2",
     "compression": "^1.7.0",
+<<<<<<< HEAD
```

Comment:
ooops you accidentally checked in a merge conflict. Please remove these bad lines, rebase this branch, do a clean `npm install` and make sure to checkin changes to the `package-lock.json`